### PR TITLE
Fix podfile dependency management

### PR DESCRIPTION
**Pull request by guperrot**
* Handle updating SDK
* Do not always bring both Analytics and Crashes

### CODE REVIEW

File: mobile-center-analytics/scripts/postlink.js [hunk 1]
```diff
@@ -14,8 +14,7 @@ return rnpmlink.ios.initMobileCenterConfig().then(function (file) {
 }).then(function (file) {
     console.log('Added code to initialize iOS Analytics SDK in ' + file);
     return rnpmlink.ios.addPodDeps([
-        { pod: 'MobileCenter', version: '0.11.0' },
-        { pod: 'RNMobileCenterShared', version: '0.8.0' }
+        { pod: 'MobileCenter/Analytics', version: '0.11.0' }
```
Comment (BretJohnson) [inline mobile-center-analytics/scripts/postlink.js]: Don't you still need to include RNMobileCenterShared here?

File: mobile-center-crashes/scripts/postlink.js [hunk 2]
```diff
@@ -14,8 +14,7 @@ return rnpmlink.ios.initMobileCenterConfig().then(function (file) {
 }).then(function (file) {
     console.log('Added code to initialize iOS Crashes SDK in ' + file);
     return rnpmlink.ios.addPodDeps([
-        { pod: 'MobileCenter', version: '0.11.0' },
-        { pod: 'RNMobileCenterShared', version: '0.8.0' }
+        { pod: 'MobileCenter/Crashes', version: '0.11.0' }
```
Comment (BretJohnson) [inline mobile-center-crashes/scripts/postlink.js]: Don't you still need to include RNMobileCenterShared here?

File: mobile-center/scripts/postlink.js [hunk 3]
```diff
@@ -7,8 +7,7 @@ return rnpmlink.ios.initMobileCenterConfig().then(function (file) {
 }).then(function (file) {
     console.log('Added code to initialize iOS Mobile Center SDK in ' + file);
     return rnpmlink.ios.addPodDeps([
-        { pod: 'RNMobileCenterShared', version: '0.8.0' },
-        { pod: 'MobileCenter', version: '0.11.0' }
+        { pod: 'RNMobileCenterShared', version: '0.8.0' }
```
Comment (BretJohnson) [inline mobile-center/scripts/postlink.js]: Don't you still need to include RNMobileCenterShared here?

File: mobile-center-push/scripts/postlink.js [hunk 4]
```diff
@@ -7,9 +7,7 @@ return rnpmlink.ios.initMobileCenterConfig().then(function (file) {
 }).then(function (file) {
     console.log('Added code to initialize iOS Push SDK in ' + file);
     return rnpmlink.ios.addPodDeps([
-        { pod: 'MobileCenter', version: '0.11.0' },
-        { pod: 'MobileCenter/Push', version: '0.11.0' },
-        { pod: 'RNMobileCenterShared', version: '0.8.0' }
+        { pod: 'MobileCenter/Push', version: '0.11.0' }
```
Comment (BretJohnson) [inline mobile-center-push/scripts/postlink.js]: Don't you still need to include RNMobileCenterShared here?

File: mobile-center-analytics/scripts/postlink.js (hunk 1 above)
Comment (guperrot) [inline mobile-center-analytics/scripts/postlink.js]: No

File: mobile-center-crashes/scripts/postlink.js (hunk 2 above)
Comment (guperrot) [inline mobile-center-crashes/scripts/postlink.js]: No

File: mobile-center/scripts/postlink.js (hunk 3 above)
Comment (guperrot) [inline mobile-center/scripts/postlink.js]: No

File: mobile-center-push/scripts/postlink.js (hunk 4 above)
Comment (guperrot) [inline mobile-center-push/scripts/postlink.js]: No

**Comment by guperrot** [comment]
@BretJohnson RNMobileCenterShared comes with javascript mobile-center module, which is a dependency of every javascript module. So it's only needed in JS mobile-center.

File: mobile-center/scripts/postlink.js (hunk 3 above)
Comment (guperrot) [inline mobile-center/scripts/postlink.js]: (also RNMobileCenterShared now depends only on MobileCenter/Core)

**Pull request merged**